recipes/zstd/all/conanfile.py

```diff
@@ -86,24 +86,25 @@ def package(self):
         rmdir(self, os.path.join(self.package_folder, "share"))
 
         if self.options.shared and self.options.build_programs:
```

Comment:
```suggestion
        if self.options.shared:
```

Programs are statically linked already, we don't need to package them anyway.

Reply:
This branch removes static libs when you have both shared & build_programs options enabled, it's correct as it is. See generate() method.